src/locales.json

```diff
@@ -70,6 +70,18 @@
       }
     }
   },
+  "categories": {
+    "ar": {
+    },
+    "de": {
+      "createPdf": "Erstelle PDF"
```

Comment:
Is "categories" the right place for this? We maybe want to display the toolbar with the PDFButton on other pages, too, not only on the categories page.